web/app/views/applications/show.html.erb

```diff
@@ -21,10 +21,16 @@
             data-id="<%= @app.id %>"
             data-state="<%= @app.creation_state %>">
 	      <%=
-	      case @app.creation_state
-	      when :CREATE_INFRASTRUCTURE_WAIT then render("shared/spinner")
-	      when :SUCCEEDED then 'Ready'
-	      when :FAILED then 'Failed to build'
+	      case @app.deletion_state
```

Comment:
It looks like the white space varies a bit here.

Reply:
I think I'll open a formatting PR soon and take care of this – these files are all indented with a mix of tabs and spaces, and different tab sizes, so I'd like to standardize that all at once.

Speaking of which, do you know any tools that will do that automatically for erb files?

Reply:
I feel your pain. I don't know any simple tools for formatting erb files but an `.editorconfig` file will help a lot.